modules/virt-importing-rhel-image-boot-source-web.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-importing-rhel-image-boot-source-web_{context}"]
+= Importing a Red Hat Enterprise Linux image as a boot source
+
+You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by specifying the URL address for the image.
+
+.Prerequisites
+* To import a RHEL image via a URL, access to the web server with the operating system image is required. For example: the Red Hat Enterprise Linux web page with images.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+
+. Click the *Templates* tab.
+
+. Identify the RHEL template that you want to use for creating a virtual machine.
+
+. Click *Add source* for the identified template.
+
+. In the *Add source to vendor template* dialog, select *Import via URL (creates PVC) from the *Boot source type*.
+
+. Under the *Import via URL (creates PVC)* field, click *RHEL cloud image list* to view the list of available KVM guest images.
```

Comment:
Just installed 4.8 to check, this still says "RHEL download page" which takes you to https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.2/x86_64/product-software 

The download list would be better referred to as available installers and images. (Just one of these per version is a KVM guest image, the rest are various ISOs and driver update disks)

Reply:
Modified this content.